spec/support/shared_contexts/rake.rb

```diff
@@ -0,0 +1,19 @@
+require 'rake'
+
+shared_context 'rake' do
+  let(:rake)      { Rake::Application.new }
```

Comment:
I don't have a strong feeling about this, but I think sometimes the extra spacing makes things more readable, so I would lean towards updating the Hound config. The setting should be:
```
Style/ExtraSpacing:
  Description: Do not use unnecessary spacing.
  Enabled: false
```
If you do add this setting, please place it in alphabetical order in `.rubocop.yml`